pkg/odo/cli/application/application.go

```diff
@@ -1,25 +1,26 @@
 package application
 
 import (
+	"encoding/json"
 	"fmt"
-	"github.com/redhat-developer/odo/pkg/odo/util/validation"
-	"github.com/redhat-developer/odo/pkg/service"
 	"os"
 	"strings"
+	"text/tabwriter"
+
+	"github.com/redhat-developer/odo/pkg/odo/util/validation"
```

Comment:
these imports are mess :-( 

ideal grouping should be stdlibs, odo imports, kubeimports, 3rd party imports